Source/EasyNetQ.Logging.Serilog/ServiceRegisterExtensions.cs

```diff
@@ -0,0 +1,38 @@
+using EasyNetQ.DI;
+using EasyNetQ.Logging;
+using EasyNetQ.Logging.Serilog;
+using S = Serilog;
+
+// ReSharper disable once CheckNamespace
+namespace EasyNetQ
+{
+    /// <summary>
+    ///     Register loggers based of Microsoft.Extensions.Logging
+    /// </summary>
+    public static class ServiceRegisterExtensions
+    {
+        /// <summary>
+        ///     Enables serilog logging support for EasyNetQ. It should be already registered in DI.
+        /// </summary>
+        /// <param name="serviceRegister">The register</param>
+        public static IServiceRegister EnableSerilogLogging(this IServiceRegister serviceRegister)
+        {
+            return serviceRegister
+                .Register(typeof(ILogger), typeof(SerilogLoggerAdapter))
+                .Register(typeof(ILogger<>), typeof(SerilogLoggerAdapter<>));
+        }
+
+        /// <summary>
+        ///     Enables serilog logging using provided Serilog.ILogger
+        /// </summary>
+        /// <param name="serviceRegister">The register</param>
+        /// <param name="logger"></param>
+        public static IServiceRegister EnableSerilogLogging(this IServiceRegister serviceRegister, S.ILogger logger)
```

Comment:
I would get rid of this overload. Serilog [registers](https://github.com/serilog/serilog-extensions-hosting/blob/dev/src/Serilog.Extensions.Hosting/SerilogHostBuilderExtensions.cs#L239) its logger at least for majority of ASP.NET Core/Hosted projects.

Reply:
What should we do for a simple console app without DI container, which wants to log via Serilog?

Reply:
If app does not use DI container then that app would not use `EnableSerilogLogging` method at all. Right?

Reply:
Why? I mean EasyNetQ has it's own internal container and it could be used for `EnableSerilogLogging` even without any container on application side.

Reply:
@sungam3r ?

Reply:
I know. I do not use the `CreateBus` method, only `RegisterBus` one with "real" external container. Nevertheless `CreateBus` accepts `Action<IServiceRegister> registerServices` argument so caller may register anything - Serilog or something else. In fact anyone who has already integrated ENQ with Serilog one way or another did it. What I try to say is that it is **not your responsibility** to accept such parameters in this API.

Reply:
Ok, let's ditch these overloads and will see.